content/en/docs/reference/glossary/namespace-sameness.md

```diff
@@ -6,4 +6,6 @@ test: n/a
 Within a multicluster mesh, [namespace sameness](https://github.com/kubernetes/community/blob/master/sig-multicluster/namespace-sameness-position-statement.md)
 applies and all namespaces with a given name are considered to be the same namespace. If multiple clusters contain a
 `Service` with the same namespaced name, they will be recognized as a single combined service. By default, traffic is
-load-balanced across all clusters in the mesh for a given service.
\ No newline at end of file
+load-balanced across all clusters in the mesh for a given service.
+
+For one or many ports within the combined service, same port value should also have same port name.
```

Comment:
```suggestion
Ports that match on _number_ must also have the same port _name_ to be considered as a combined service port.```

Reply:
Thanks for reply. I updated, please have a look.